src/Surfnet/StepupMiddlewareClientBundle/Identity/Service/SecondFactorService.php

```diff
@@ -204,6 +204,35 @@ public function searchVerified(VerifiedSecondFactorSearchQuery $query)
         return $secondFactors;
     }
 
+    /**
+     * @param VerifiedSecondFactorSearchQuery $query
+     * @return VerifiedSecondFactorCollection
+     * @throws AccessDeniedToResourceException When the consumer isn't authorised to access given resource.
+     * @throws InvalidResponseException When the API responded with invalid data.
+     * @throws ResourceReadException When the API doesn't respond with the resource.
+     * @throws MalformedResponseException When the API doesn't respond with a proper response.
+     */
+    public function searchVerifiedWithoutAuthorizationContext(VerifiedSecondFactorSearchQuery $query)
```

Comment:
I think this could be better renamed to be more descriptive for example `searchOwnTokensForSelfservice` or something like that. Because it can only be used by SS and is only used to list own tokens we should drop complexity if we can.

Reply:
Good point, I'm going for: `searchOwnVerified`. Adding `tokens` and `self service` did not make the cut, as I tried to stick as close to the naming of the 'sister' method.

Reply:
I think it would be safer to use a custom `OwnVerifiedSecondFactorSearchQuery` instead of the `VerifiedSecondFactorSearchQuery` currently being used.

Reply:
Yes good suggestion, further separated the logic by adding an additional query.